Source/Urho3D/IO/Serializer.cpp

```diff
@@ -285,10 +287,24 @@ bool Serializer::WriteVariantData(const Variant& value)
         return WriteBuffer(value.GetBuffer());
 
         // Serializing pointers and custom values is not supported. Write null
+    case VAR_CUSTOM_STACK:
+    {
+        if (const auto* ptr = value.GetCustomPtr<SharedPtr<Serializable>>())
```

Comment:
It could be
`const Serializable* object = value.GetCustom<SharedPtr<Serializable>>()`
here and in other places